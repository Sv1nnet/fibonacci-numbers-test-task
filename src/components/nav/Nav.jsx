import React from 'react';

import './nav.scss';

const Nav = () => (
  <nav className="light-green darken-3">
    <div className="nav-wrapper">
      <h4 className="center nav-text"><a href="/">Fibonacci numbers</a></h4>
    </div>
  </nav>
);

export default Nav;
