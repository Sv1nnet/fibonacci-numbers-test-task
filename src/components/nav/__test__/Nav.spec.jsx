import {
  it,
  describe,
  expect,
} from '@jest/globals';
import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import Nav from '../Nav';

configure({ adapter: new Adapter() });

describe('Test Nav', () => {
  it('Shoul match snapshot', () => {
    const component = shallow(<Nav />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
