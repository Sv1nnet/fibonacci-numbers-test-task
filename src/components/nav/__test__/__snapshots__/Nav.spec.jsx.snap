// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Nav Shoul match snapshot 1`] = `
<nav
  className="light-green darken-3"
>
  <div
    className="nav-wrapper"
  >
    <h4
      className="center nav-text"
    >
      <Link
        to="/"
      >
        Fibonacci numbers
      </Link>
    </h4>
  </div>
</nav>
`;
